app/assets/json/translations/react/en.json

```diff
@@ -49,6 +49,10 @@
   "listings.additionalEligibilityRules.title": "Additional Eligibility Rules",
   "listings.additionalInformation.header": "Additional Information",
   "listings.additionalInformation.subheader": "Required documents and selection criteria",
+  "listings.allUnitsReservedFor.Accessible Units Only": "All units reserved for people who need accessibility features",
```

Comment:
not sure if it's overboard, but it might be a good idea to reference these with the RESERVED_COMMUNITY_TYPES `consts` since we're accessing these dynamically